dsl_parser/tests/test_get_attribute.py

```diff
@@ -354,48 +344,44 @@ def _test_process_attribute_scaling_group_ambiguity_resolution(
                     for r in node_instance.get('relationships', [])],
             })
 
-        def get_node_instances(node_id):
-            return node_to_node_instances[node_id]
+        class Storage(object):
```

Comment:
I think u can generalize the storage so it can be more easy to write test with this